synthtool/gcp/templates/python_library/noxfile.py.j2

```diff
@@ -37,7 +37,7 @@ def lint(session):
     Returns a failure if the linters find linting errors or sufficiently
     serious code quality issues.
     """
-    session.install("flake8", "black", *LOCAL_DEPS)
+    session.install("flake8", "black==19.3b0", *LOCAL_DEPS)
```

Comment:
ISTM it would be cleaner to use a module-scope constant, e.g.:

```python
BLACK_VERSION = "black==19.3b0"
...
    session.install("flake8", BLACK_VERSION, *LOCAL_DEPS)
```

Reply:
*shrugs* sure.